### ISSUE TITLE
00-Before-The-Workshop

### ISSUE THREAD

**Issue by ldecicco-USGS** [USGS-R/introR#42]
Add instructions to have TeX tools installed. Maybe delete before the workshop intro? We want to re-use this for Service Desk installation. Also, add description tag somehow.

**Comment by jread-usgs** [comment]
make sure you click "instal R as an admin" otherwise RStudio can't find R (windows)

**Comment by ldecicco-USGS** [comment]
Which leads me to think we *do* need to have them change permissions on that folder. like the original instructions.

**Comment by lawinslow** [comment]
No, we don't. If you don't change permissions, R will just figure out it can't write to Program Files library area, and switch to your user library directory. But it *is* important to install R as administrator.

**Comment by ldecicco-USGS** [comment]
Not sure if this should be included here or somewhere else. The IT guys could certainly verify there is a CRAN mirror (half of the class here had the default RStudio cloud mirror, the other half had no mirror), then add GRAN:

```r
source("http://owi.usgs.gov/R/add_gran_repo.R")
```

I think the user should then be asked to install the packages before the workshop:
```r
install.packages(c("dataRetrieval","gsIntroR","dplyr","ggplot2","smwrBase","smwrGraphs","smwrData","EGRET"))
```

And we need to include a screenshot of the pop-up window that asks if it is OK to create a user library. I think it's a bug in RStudio myself, but that pop-up doesn't pop up over RStudio...it is behind it, so no one notices unless they look down and notice there's a weird window now in their taskbar (probably just a Windows thing). Then, if the click anything...the default is 'no', so a library isn't created.
